conf/locale/ru/LC_MESSAGES/django.po

```diff
@@ -15798,11 +15798,18 @@ msgid ""
 "engaging, high-quality {platform_name} courses. Note that you will not be "
 "able to log back into your account until you have activated it."
 msgstr ""
+"Привет!"
+"Используйте нашу ссылку, чтобы активировать свою учетную запись и получить доступ к курсам MAMA."
+"MAMA - это обучение от ведущих преподавателей и высокое качество полученных знаний онлайн."
+"Жми на ссылку:"
 
 #: lms/templates/emails/activation_email.txt
 msgid "Enjoy learning with {platform_name}."
 msgstr ""
-
+"Важно:"
+"Вы не сможете снова войти в свою учетную запись, пока не активируете ее."
+"Если Вам понадобится помощь, пишите нам на электронною почту academy@mama.mobytize.com"
```

Comment:
Is this email really supposed to be hardcoded here?

Reply:
I didn't find any variable for using instead. And thats why I hardcoded this here.

Reply:
There is an english line "If you need help, please use our web form at {support_url} or email " - isn't it the same?